topicExtractor.py

```diff
@@ -196,6 +198,34 @@ def fitTopicModel(self):
                 f"Failed to send message at max limit of {self.callMaxLimit} times."
             )
             return False
+        
+    
+    def useKMeans(self, vectorizerModel, docs, n_clusters=3):
```

Comment:
This function is called when `vectorizerModel` is `None` as well. So I will it helps to logs message if this function is called with model object or not?

Reply:
I guess if the the warn message is always thrown from L227 here in case of `vectorizerModel = None` then separate log message is not needed

Reply:
At L1559, there is a message logged if the function is called with no Vectorizer. 
`logging.warn("Retrying without Vectorizer model.")`
Shall I add another log message within the function itself?

Reply:
No Need if there is somewhere it is logged.